x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/hooks/use_monitor_errors.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEsSearch } from '@kbn/observability-plugin/public';
+import { useParams } from 'react-router-dom';
+import { useMemo } from 'react';
+import { Ping } from '../../../../../../common/runtime_types';
+import {
+  EXCLUDE_RUN_ONCE_FILTER,
+  SUMMARY_FILTER,
+} from '../../../../../../common/constants/client_defaults';
+import { SYNTHETICS_INDEX_PATTERN } from '../../../../../../common/constants';
+import { useSyntheticsRefreshContext } from '../../../contexts';
+import { useGetUrlParams } from '../../../hooks';
+
+export function useMonitorErrors(monitorIdArg?: string) {
+  const { lastRefresh } = useSyntheticsRefreshContext();
+
+  const { monitorId } = useParams<{ monitorId: string }>();
```

Comment:
Will only work for non-project monitors. :(

I'm tempted to let this slide and have the ultimate fix for that come in sweeping changes in the issue created to simpifying logic for project monitors.